pages/index.py

```diff
@@ -34,27 +34,42 @@ def slider_item(title, text, active=""):
             slider_item(_("Avoid Censorship"), md(_("""
 Freenet is a platform for censorship-resistant communication and publishing.
 It helps you to remain anonymous, and communicate without fear.
+
+*'Daddy, where were you when they took freedom of the press away from the Internet?'*  
+— Mike Godwin ([about Freenet](about.html))
 """)), "active"),
             slider_item(_("Host a Website"), _("""
 Need a website nobody can take over? That is hosted for free? That is very
 resistant to attacks? Publish it on Freenet!
+
+*'I never had to worry no more about security of my sites. With Freenet I am Free.'* — Unkwon  
```

Comment:
What service / platform was this user on? We could make it clearer that it's a username and not a typo for unknown with "Reddit user Unkwon".